main.py

```diff
@@ -21,10 +21,10 @@ def main():
     run(['mkdir', '-p', settings['tmp_pred_path']])
     run(['rm', '-rf', os.path.join(settings['tmp_pred_path'], '*')])
     cmd = [
-        'singularity', 'run', '--nv', '--pwd', settings['run']['pwd'],
+        'singularity', 'run', '--nv',
         '--bind', f"{settings['tmp_pred_path']}:{settings['run']['pred_path']},"
                   f"{settings['val_rgb_path']}:{settings['run']['rgb_path']}",
-        os.path.join('/tmp', f'fishyscapes_pr_{pr_id}.simg')
+        os.path.join('/tmp', f'fishyscapes_pr_{pr_id}.simg') + " python3 /workspace/test.py --weight_path=/workspace/test.pth"
```

Comment:
This is a problem, you cannot just change the run-command in the CI script since your container now is incompatible with all my test scripts ;)

Please just add the run command to the %runscript part of your singularity definition file and rebuild the container: https://apptainer.org/docs/user/latest/definition_files.html#runscript